oslabs-beta/Kubiquity
# about section

WHAT DOES THIS PR DO? 
1. Adds in a new frontend component
2. Adds about section. 

HOW TO TEST THIS PR:
1. I added a new package, so  'run gh pr checkout 27'  'npm install'  'npm run dev' 
2. Scroll to bottom of page. Check if all information is up to date. 

ADDITIONAL NOTES: 
No styling to this component at this time.  Will add styles on next PR.
